packages/ember-htmlbars/tests/compat/view_helper_test.js

```diff
@@ -71,14 +71,15 @@ QUnit.test('using the view helper with a string (inline form) fails assertion [L
   const ViewClass = EmberView.extend({
     template: compile('fooView')
   });
-  registry.register('view:foo', ViewClass);
+  owner.register('view:foo', ViewClass);
 
   ignoreAssertion(function() {
     component = EmberComponent.extend({
-      layout: compile('{{view \'foo\'}}'),
-      container
+      layout: compile('{{view \'foo\'}}')
     }).create();
 
+    setOwner(component, owner);
```

Comment:
Unfortunately many of these tests artificially create the view/component instances (i.e. by creating them directly instead of lookup them up from a container), so they do not have the proper injections initially.  We could  update to export the owner symbol and set directly in `.create`, but a future refactoring is already planned to replace all of these `View.create()` calls with something very close to the ember-qunit API.